system-addon/test/unit/common/Actions.test.js

```diff
@@ -1,10 +1,26 @@
 const {
+  actionTypes: at,
   actionCreators: ac,
   actionUtils: au,
   MAIN_MESSAGE_TYPE,
-  CONTENT_MESSAGE_TYPE
+  CONTENT_MESSAGE_TYPE,
+  UI_CODE,
+  BACKGROUND_PROCESS,
```

Comment:
I disagree in this case.

When constants refer to something external to the module, such as a DOM event or a URL, having a separate source of truth is useful to test for errors/typos in the constants themselves.

These constants, on the other hand, do not refer to anything external to the module; they exist only in the context of the module itself. They're only created for *internal* consistency.  For that reason, there is no benefit to creating a separate, external source of a truth.